.github/workflows/browser.yml

```diff
@@ -64,6 +64,7 @@ jobs:
           key: ${{ inputs.submodule-cache-key }}
 
       - run: npm run install-browser-deps
+      - run: npx playwright install --with-deps
```

Comment:
Why did you change this away from using the npm script?

Reply:
I have absolutely no idea what happened here. 😬 Didn't you want to update these kind of things, or was this not "on the list"? Should we still do?

Reply:
I think this line should be removed?

This is the `install-browser-deps` npm script:

`"install-browser-deps": "npm install playwright@1.15.1 @vitest/browser@3.0.8"`

(Unless the `npx` does something different and the `--with-deps` flag should be added?)

Also, playwright is not in package.json, is this intended? `webdriverio` was in our package, but playwright is not. (Weird, I'm sure a `npm i` installed playwright for me yesterday)

Reply:
Update: ah, just recalled, so I added this line accidentally since I didn't bring together with the `npm run install-browser-deps` task on first iteration, should be safe to remove.

See this comment https://github.com/ethereumjs/ethereumjs-monorepo/pull/3957#issuecomment-2772269452 from (in response to) @acolytec3. We should put *these* kind of things really in the dev docs on the side line (but culturally integrated into normal team-workflow), so the things which always gets a re-ask again and again.

(and if the question arises since one can't have the full docs in mind it is at least possible to quickly point to the respective section if at least one person remembers... 😋)